receptai.api/Controllers/SubfoodditUserController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using receptai.api.Interfaces;
+using System.Threading.Tasks;
+
+namespace receptai.api.Controllers;
+
+[Route("api/subfooddit-users")]
+[ApiController]
+public class SubfoodditUserController : ControllerBase
+{
+    private readonly ISubfoodditRepository _subfoodditRepository;
+
+    public SubfoodditUserController(ISubfoodditRepository subfoodditRepository)
+    {
+        _subfoodditRepository = subfoodditRepository;
+    }
+
+    [HttpPost("{userId:int}")]
+    public async Task<IActionResult> AddUser(int subfoodditId, int userId)
+    {
+        var success = await _subfoodditRepository.AddUserToSubfooddit(subfoodditId, userId);
+        if (!success)
+        {
+            return NotFound("Subfooddit or user not found.");
+        }
+
+        return Ok();
+    }
+
+    [HttpDelete("{userId:int}")]
+    public async Task<IActionResult> RemoveUser(int subfoodditId, int userId)
+    {
+        var success = await _subfoodditRepository.RemoveUserFromSubfooddit(subfoodditId, userId);
+        if (!success)
+        {
+            return NotFound("Subfooddit or user not found.");
+        }
+
+        return NoContent();
+    }
+
+    [HttpGet("by-user/{userId}")]
+    public async Task<IActionResult> GetSubfoodditsByUserId(int userId)
+    {
+        var subfooddits = await _subfoodditRepository.GetSubfoodditsByUserId(userId);
+        if (subfooddits == null || subfooddits.Count == 0)
+        {
+            return NotFound("No subfooddits found for the provided user ID.");
+        }
+        return Ok(subfooddits);
+    }
```

Comment:
@Andrius-Sukys Sprint PO - should user B be able to query what subfoodits user A is subscribed to?

Reply:
When we talked during our weekly standups, we kind of agreed that users would be able to access other user's profiles, so I guess they would be able to see what that user has subscribed to? But our PO will decide if this is the right way to get that info :)

Reply:
@R0PK3 since user profiles were indeed in scope of our product, I believe it makes sense to include this relevant info in the profile of a given user - otherwise, only recipes would appear in one's profile which makes it practically the same as the main feed of the application. In order to differentiate, I think including subfooddits in the profiles makes sense.